### ISSUE TITLE
"Error occurred while the 'sparklyr' package was updating the RStudio Connections pane"

### ISSUE THREAD

**Issue by edgararuiz** [mlverse/pysparklyr#44]
Whenever there is additional information in the Databricks Host value, as in, any parameters, a Connections pane issue is raised. This is because the Host needs to be the base url, with out any parameters

```r
> Sys.setenv("DATABRICKS_HOST" = "https://rstudio-partner-posit-default.cloud.databricks.com/o=notvalid")
> sc <- spark_connect(
+   master = Sys.getenv("DATABRICKS_HOST"),
+   cluster_id = Sys.getenv("DATABRICKS_CLUSTER_ID"),
+   token = Sys.getenv("DATABRICKS_TOKEN"),
+   method = "databricks_connect"
+ )
# ! Retrieving version from cluster '0914-200113-rc8v8keb'
# ! A Python environment with a matching version was not found
# * Will attempt connecting using 'r-sparklyr-databricks-14.0'
# * To install the proper Python environment use: pysparklyr::install_databricks(version = "")
# ✔ Using the 'r-sparklyr-databricks-14.0' Python environment (/Users/edgar/.virtualenvs/r-sparklyr-databricks-14.0/bin/python)
# An error occurred while the 'sparklyr' package was updating the RStudio Connections pane:
# Error in if (spark_connection_is_yarn(scon)) {: argument is of length zero
# Error in     actions <- c(actions, list(YARN = list(icon = file.path(icons, : argument is of length zero
# Error in         "yarn-ui.png"), callback = function() {: argument is of length zero
# Error in         browse_url(spark_connection_yarn_ui(scon)): argument is of length zero
# Error in     }))): argument is of length zero
# Error in }: argument is of length zero
# If necessary, these warnings can be squelched by setting `options(rstudio.connectionObserver.errorsSuppressed = TRUE)`.
```

**Solution** - Remove any parameters, and leading forward slashes, from the URL.  In the package, we'll check for that going forward and provide clearer warnings

**If you do not have an invalid host, or, if after correcting the host, please comment on this issue to let us know**

**Comment by edgararuiz** [comment]
Addressed in #46

**Comment by edgararuiz** [comment]
New version now on CRAN: https://github.com/mlverse/pysparklyr/tree/v0.1.1
